errors.js

```diff
@@ -0,0 +1,20 @@
+const ES6Error = require("es6-error");
+
+exports.onlyInstancesOf = function (errorType, handler) {
+  return (error) => {
+    if (error instanceof errorType) {
+      return handler(error);
+    // eslint-disable-next-line no-else-return
+    } else {
+      console.log(error.stack);
+      process.exit(1);
```

Comment:
@saintsebastian `process.exit(1)` seems a bit extreme here in a reusable helper, it will make the application to potentially exit immediatelly everywhere we use this helper, we should just `throw error;` here and let the caller handle the re-throw exception as needed (e.g. logging the error stack and `process.exit(1)`).